scripts/helper.ts

```diff
@@ -48,16 +48,22 @@ function typeMap(type: string, impHolder: Map<string, Set<string>>): string {
     case 'RightHolder[]':
     case 'CodeType':
     case 'CodeType[]':
+    case 'execution_code_data':
       typeName = formatName(type);
       addImport(impHolder, `./${typeName}`, typeName);
       break;
+    case 'execution_code_data[]':
```

Comment:
This does not look good. Doubt checking, is there any better solution?

Reply:
I have changed it, if you have any suggestion please tell me!.